### PR TITLE
Show suborg name in the user dropdown for suborg user's myaccount login

### PR DESCRIPTION
**Pull request by pavinduLakshan**
### Purpose

$subject

### Related Issues
- N/A

### Related PRs
- N/A

### Checklist
- [ ] e2e cypress tests locally verified.
- [x] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [x] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [x] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/myaccount/src/components/shared/header.tsx [hunk 1]
```diff
@@ -112,6 +112,12 @@ export const Header: FunctionComponent<HeaderPropsInterface> = (
     const tenantName: string = useSelector(
         (state: AppState) => state.authenticationInformation.tenantDomain
     );
+
+    // B2B organization name if the user has logged in through B2B organization.
+    const orgName: string = useSelector(
+        (state: AppState) => state.authenticationInformation.orgName
```
Comment (dasuni-30) [inline apps/myaccount/src/components/shared/header.tsx:118]: Let's add null checks

File: apps/myaccount/src/components/shared/header.tsx [hunk 2]
```diff
@@ -271,16 +277,19 @@ export const Header: FunctionComponent<HeaderPropsInterface> = (
...
      */
-    const resolveOrganizationLabel = (): ReactElement => {
-        const organization: string =
-            tenantName == "carbon.super"
-                ? commonConfig.header.organization.replace("{{productName}}", productName)
-                : tenantName;
+    const resolveOrganizationLabel =  (): ReactElement => {
+        let organizationName: string = tenantName;
+
+        if (tenantName === "carbon.super") {
```
Comment (dasuni-30) [inline apps/myaccount/src/components/shared/header.tsx:283]: Shall we add a constant for this

Comment (pavinduLakshan) [reply apps/myaccount/src/components/shared/header.tsx]: Fixed with [2d47cf8](https://github.com/wso2/identity-apps/pull/6466/commits/2d47cf85f055a223fc2874c0d2c09a5f5aa1f628)

File: apps/myaccount/src/components/shared/header.tsx (hunk 1 above)
Comment (pavinduLakshan) [reply apps/myaccount/src/components/shared/header.tsx]: Fixed with f851aa2

**Review by DonOmalVindula — APPROVED** [review]

**Pull request merged**